help/en/package/jmri/jmrit/operations/Operations.shtml

```diff
@@ -260,20 +260,29 @@
     
     <a name="Display"></a>
     <h3>Display and Help</h3>
-    
-    To adjust the display font size or appearance of the JMRI windows, select "Preferences.." found under
-    "Edit" in the main window menu bar. A new window will appear with options to make layout connections, change
-    the "Display" appearance and font size, and many other settings including the "Railroad Name". Most
-    JMRI windows have as part of the menu bar the "Help" option. Select "Window Help.." to get information
-    about the window you're using. 
-    
+
```

Comment:
It would be even better to add this to the [Preferences help](http://jmri.sourceforge.net/help/en/package/apps/TabbedPreferences.shtml) and just add a link from this page. That way, we don't have to maintain the same info in two places.